python/dasclient/dasclient.py

```diff
@@ -194,6 +194,11 @@ def time_converter(t):
             raise DasClientPermissionDenied(reason)
 
         message = f"provider_key: {self.provider_key}, service: {self.service_root}, path: {path},\n\t {response.status_code} from ER. Message: {reason} {response.text}"
+
+        if response.status_code == 504:  # gateway timeout
+            self.logger.error(f"{message}")
```

Comment:
Here I think it would be good to include extra, containing the message details -- then they'll be available for filtering/metrics/alerting when applicable.

Reply:
@jgoodheart are you interested in adding the extra={} arg here?